proto/mixer.proto

```diff
@@ -374,23 +374,23 @@ service Mixer {
   rpc PropertyValues(datacommons.v1.PropertyValuesRequest)
```

Comment:
@juliawu PTAL the URI pattern change here.

Reply:
LGTM!